src/renderers/WebGLRenderer.js

```diff
@@ -1286,8 +1287,15 @@ function WebGLRenderer( parameters = {} ) {
 		_this.setRenderTarget( _transmissionRenderTarget );
 		_this.clear();
 
+		// Turn off the features which can affect the frag color for opaque objects pass.
+		// Otherwise they are applied twice in opaque objects pass and transmission objects pass.
+		const currentToneMapping = _this.toneMapping;
+		_this.toneMapping = NoToneMapping;
```

Comment:
Oh! Actually... I guess we should also set `outputEncoding` to `LinearEncoding`?

Reply:
No because we use renderTarget texture's encoding as output encoding if `currentRenderTarget` isn't `null`.

https://github.com/mrdoob/three.js/blob/r129/src/renderers/webgl/WebGLPrograms.js#L174

If we start to copy the framebuffer we will need to convert from sRGB to linear and from tone mapping to none in the shader if they are set.